curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/65549f90cf78131c96ebcf28.md

```diff
@@ -0,0 +1,51 @@
+---
+id: 65549f90cf78131c96ebcf28
+title: Step 56
+challengeType: 20
+dashedName: step-56
+---
+
+# --description--
+
+When coding, readability is key. Comments serve as effective notes, explaining the logic behind your code. They prove valuable when returning to a project after some time and also aid your colleagues in understanding the code.
+
+In Python, you can write a comment using a `#`. Anything that comes after the `#` won't be executed.
+
+Before your `if` statement, add a comment saying `Append space to the message`.
```

Comment:
I understand the need to introduce how to use comments. However used examples are focused on _what_ is in the code - not _why_, they don't add anything that cannot be read from code itself almost effortlessly.

Some idea for making comment justified:
```suggestion
Before your `if` statement, add a comment saying `Spaces are not encrypted in the message`.
```

Reply:
I'm going to keep this and the other hint-related conversation unresolved for now. Since the overall structure of this project is fine, I will prioritize the other projects and come back to hints later on. Thanks for your suggestions.